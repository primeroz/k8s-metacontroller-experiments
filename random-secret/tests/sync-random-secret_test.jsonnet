local test = import '../manifests/sync-random-secret.jsonnet';

local data = {
  parent: {
    apiVersion: 'primeroz.xyz/v1',
    kind: 'RandomSecret',
    metadata: {
      name: 'test1',
    },
    spec: {
      length: 10,
      secretName: 'secret1',
    },
  },
};


test(data)
