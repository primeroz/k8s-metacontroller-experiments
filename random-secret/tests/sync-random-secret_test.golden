{
   "attachments": [
      {
         "apiVersion": "v1",
         "data": {
            "value": "dGVzdA=="
         },
         "kind": "Secret",
         "metadata": {
            "labels": {
               "app": "test"
            },
            "name": "secret1"
         }
      }
   ]
}
